app/clients/smtp/aws.py

```diff
@@ -0,0 +1,207 @@
+import boto3
+from flask import current_app
+
+import hmac  # required to compute the HMAC key
+import hashlib  # required to create a SHA256 hash
+import base64  # required to encode the computed key
+import time
+
+
+def smtp_add(name):
+    ses_client = boto3.client('ses', region_name=current_app.config["AWS_REGION"])
+    r53_client = boto3.client('route53', region_name=current_app.config["AWS_REGION"])
+    iam_client = boto3.client('iam', region_name=current_app.config["AWS_REGION"])
+
+    # name = name + '.m.' + current_app.config["NOTIFY_EMAIL_DOMAIN"]
+    name = name + '.m.' + 'notification.alpha.cdssandbox.xyz'
```

Comment:
Assume there's no way to have some string termination in either the strcat, or in amazon's parsing of the domain, that would allow for a domain that's *not* a sub of the cdssandbox.xyz... May be worth a quick check.